modules/xfem/test/tests/switching_material/tests

```diff
@@ -20,6 +20,16 @@
     design = 'XFEMCutSwitchingMaterial.md CutSubdomainIDAux.md'
     issues = '#17041'
   []
+  [one_cut_moving_using_mesh_cut]
+    type = Exodiff
+    input = one_cut_moving_using_mesh_cut.i
+    exodiff = 'one_cut_moving_using_mesh_cut_out.e one_cut_moving_using_mesh_cut_out.e-s002 one_cut_moving_using_mesh_cut_out.e-s003 one_cut_moving_using_mesh_cut_out_cut.e'
+    map = false
+    unique_id = true
+    requirement = 'The XFEM module shall provide an ability to switch materials for one moving cut using mesh cut userobject.'
```

Comment:
The XFEM module shall allow switching materials based on the side of an interface for a single moving cut defined using a mesh as the cutting object.